ops/cli.py

```diff
@@ -279,7 +280,41 @@ def check_git_tag(config: WflInstanceConfig) -> None:
         exit(1)
 
 
+def get_git_commits_since_last_tag(config: WflInstanceConfig) -> None:
+    "Read commit messages since last tag, store to config and print."
+    command = 'git log --pretty=format:"%s" $(git describe --tags --abbrev=0 @^)..@'
+    info("=>  Reading commit messages from git log")
+    lines = subprocess.check_output(command, shell=True, encoding="utf-8").strip().split("\n")
+    info("=>  Markdown-rize log messages")
+    current_changelog = "\n".join([f"- {line}" for line in lines])
+    config.current_changelog = current_changelog
```

Comment:
Should/could we do a regex here to replace `#123` with a link to that PR?

Reply:
Sounds like a good idea, I can try to see if it's feasible with simple regex.

Reply:
Pushed new changes, I'm no expert on `regex`, lemme know if that's over comlicated! @jack-r-warren